doc/source/whatsnew/v0.24.0.txt

```diff
@@ -626,7 +626,7 @@ MultiIndex
 
 - Removed compatibility for :class:`MultiIndex` pickles prior to version 0.8.0; compatibility with :class:`MultiIndex` pickles from version 0.13 forward is maintained (:issue:`21654`)
 - :meth:`MultiIndex.get_loc_level` (and as a consequence, ``.loc`` on a :class:``MultiIndex``ed object) will now raise a ``KeyError``, rather than returning an empty ``slice``, if asked a label which is present in the ``levels`` but is unused (:issue:`22221`)
--
+- Fix `TypeError` in Python 3 when creating `MultiIndex` in which some columns are of object type, e.g. when labels are tuples  (:issue:'15457')
```

Comment:
Apart from my comment on ``object type`` ( https://github.com/pandas-dev/pandas/pull/22072/review_comment/create#discussion_r209218334 ), you have lost the ":class:" before "`MultiIndex`".